site/blog/2022/deploy-ghost-ec2-advanced.mdx

```diff
@@ -0,0 +1,187 @@
+---
+slug: deploy-ghost-ec2-advanced
+title: Advanced configuration for deploying Ghost on EC2
+date: 2022-12-28
+authors: [yrobla]
+---
+
+# Advanced configuration for deploying <a href="https://ghost.org/">Ghost</a> on EC2
```

Comment:
the other post is fairly short. I'm curious, why did you opt to split them up?

Reply:
I did it to isolate what was the basic setup vs more advanced config, to avoid a very long/complex post. But if you think it's better i can add this to the initial ghost blogpost.